### DISCUSSION TITLE
modals inside modals laravel backpack

### DISCUSSION THREAD

**Discussion by soufyaneyassin** [Laravel-Backpack/community-forum#290] [Q&A (Help)]
hey everyone, i have question about inline create operation in laravel backpack : is there possible way to create modals inside the inline create operation? i mean it's like "inline create" inside inline create ?

**Comment by pxpm** [comment]
Hello @soufyaneyassin 

I will leave here the reply I gave on gitter so others that might be interested could read it.

We decided to not follow that route (for now), technically it should be possible, but we settle for having a regular modal working than pursuit the modal over modal over modal. 

We disabled that feature here:

https://github.com/Laravel-Backpack/CRUD/blob/449ec5f21931ad507864d81d79e4d1ebd711a433/src/resources/views/crud/fields/relationship/fetch_or_create.blade.php#L63

Basically we setup an `$inlineCreate = true;` variable when the modal open, if that variable is true we don't setup the `+ Add` button in the inline create fields in the modal.

Also here: 

https://github.com/Laravel-Backpack/CRUD/blob/449ec5f21931ad507864d81d79e4d1ebd711a433/src/resources/views/crud/fields/relationship.blade.php#L21 

I really never tested if it works, I guess it could work out of the box, just give it a try and let us know the results.

Best,
Pedro

**Reply by soufyaneyassin** [reply]
it helps to show the "+Add" button in the modal, but when i click it doesn't show me the other modal, maybe it's something related to the inline_create_modal page? :
![image_2021-05-05_143733](https://user-images.githubusercontent.com/49349846/117158986-6d537480-adaf-11eb-9e32-78a81e8c0c67.png)

**Reply by pxpm** [reply]
Probably some javascript conflict with modal id's or something like that.
